print_nanny_webapp/alerts/models.py

```diff
@@ -15,13 +17,21 @@
 from rest_framework.renderers import JSONRenderer
 from anymail.message import AnymailMessage
 from asgiref.sync import async_to_sync
+from discord import Client as discordClient
 
 import stringcase
 
 from print_nanny_webapp.utils.fields import ChoiceArrayField
 
 User = get_user_model()
 logger = logging.getLogger(__name__)
+discord = discordClient()
```

Comment:
We should initialize the DiscordClient and run its event loop in a separate worker process. I'd recommend using Channels for this. 

https://channels.readthedocs.io/en/stable/topics/worker.html#worker-and-background-tasks

Here's a rough pass on how I'd try this:

1. Create a `print_nanny_webapp/alerts/consumers.py` module. `consumers.py` is the defacto name for Channels consumer classes, similar to how `models.py` holds Django models.

2. Create a DiscordConsumer class
```python
# consumers.py
import asyncio
from channels.consumer import AsyncConsumer
from discord import Client as DiscordClient

# initialize client on the main thread's event loop
loop = asyncio.get_running_loop()
discord_client = DiscordClient(loop=loop)

class DiscordConsumer(AsyncConsumer):
    # receives { "type": "trigger.alert" } messages from channel layer (Redis)
    async def trigger_alert(self, message):
        # handle message w/ discord_client accessible at module variable scope
```
```python
from channels.layers import get_channel_layer
# models.py

class RemoteControlCommandAlert:
    def trigger_discord_alert(self, data):

        discord_settings = DiscordMethodSettings.objects.get(user=self.user)
        channels = discord_settings.channel_ids
        users = discord_settings.user_ids
        # this will broadcast a message across the "channel layer" (Redis)
        async_to_sync(channel_layer.group_send){
            "discord", { "type": "trigger.alert", "data": { ... JSON serializable string } 
        }
```

3. Register a `ChannelNameRouter` in `asgi.py` 
Ref: https://channels.readthedocs.io/en/stable/topics/worker.html#receiving-and-consumers

4. Add another container runtime to docker-compose `local.py` with command:

```
$ python manage.py runworker discord
```

Let me know if that doesn't make sense!